.travis.yml

```diff
@@ -1,13 +1,17 @@
 ---
 language: ruby
-cache: bundler
-bundler_args: --without system_tests development
-script: bundle exec rake test SPEC_OPTS='--format documentation'
-sudo: false
-matrix:
-  fast_finish: true
+
+install:
+  - wget 'https://pm.puppetlabs.com/cgi-bin/pdk_download.cgi?dist=ubuntu&rel=16.04&arch=amd64&ver=latest' -q -O pdk.deb && sudo dpkg -i pdk.deb && rm pdk.deb
+
+stages:
+  - validate
+  - test
+
+jobs:
   include:
-  - rvm: 2.1.9
-    env: PUPPET_GEM_VERSION="~> 4.0" STRICT_VARIABLES="yes"
-notifications:
-  email: false
+    - stage: validate
+      script: pdk validate 2> /dev/null
+
+    - stage: test
+      script: pdk test unit 2> /dev/null
```

Comment:
The redirection of the progress output may not be needed anymore (or be replaced by a [command line options](https://puppet.com/docs/pdk/1.x/pdk_reference.html#reference-432)) for the latest versions of the PDK. But that can also be done once this PR is merged.